com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -1569,12 +1592,15 @@ RTHandle Allocator(string id, int frameIndex, RTHandleSystem rtHandleSystem)
                 );
             }
 
-            next = camera.GetCurrentFrameRT((int)HDCameraFrameHistoryType.TemporalAntialiasing)
-                ?? camera.AllocHistoryFrameRT((int)HDCameraFrameHistoryType.TemporalAntialiasing, Allocator, 2);
-            previous = camera.GetPreviousFrameRT((int)HDCameraFrameHistoryType.TemporalAntialiasing);
+            int historyType = (int)(postDoF ?
+                HDCameraFrameHistoryType.TemporalAntialiasing : HDCameraFrameHistoryType.TemporalAntialiasingPostDoF);
```

Comment:
Kind of related, but this allocates effectively another history buffer, we are a bit struggling on memory at the moment, I am not sure if we can, but it'd be nice to deallocate the history if we had it allocated but we are not using postDoF anymore (Or alternatively, add an additional asset setting for this dof so that we never allocate this buffer if users doesn't need it)

Reply:
(I would look for the API, but am doing the review in between things, I am sure there is a way to release an history target)

Reply:
I have added a clean-up call, but of course the cleanest and more general way to deal with this stuff is a render graph.

Reply:
Yup, but rendergraph at the moment still takes history from outside and imports them in so the issue would still be present. 

Cool for the cleanup!